test/test_libs.py

```diff
@@ -425,6 +427,87 @@ def test_jumanji_consistency(self, envname, batch_size):
                 )
 
 
+@pytest.mark.skipif(not _has_brax, reason="brax not installed")
+@pytest.mark.parametrize("envname", ["fast"])
+class TestBrax:
```

Comment:
None of the tests was run
We should create a dedicated pipeline (see how we did for jumanji in .circleci)